scripts/pack-packages.mjs

```diff
@@ -44,7 +44,7 @@ Promise.all(
     function pack() {
       return new Promise((resolve) => {
         exec(
-          `pnpm pack --pack-destination="${path.join(root, 'dist').replace(/\\/g, '\\\\')}"`,
+          `pnpm pack --pack-gzip-level=0 --pack-destination="${path.join(root, 'dist').replace(/\\/g, '\\\\')}"`,
```

Comment:
`pnpm pack` got very slow for the ~100MB standalone package. Since we don't care about compression here (it's only needed for running our integration tests) we decided to disable gzip compression to significantly speed up the build.